recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.5.7" %}
+
+package:
+  name: pyjulia
+  version: {{ version }}
+
+source:
+  - url: https://files.pythonhosted.org/packages/33/90/ba856cddadd923c94b7ce07dd3623d37cfea22b815076f0c70a47d867a3e/julia-{{ version }}.tar.gz
+    sha256: f24b9788f3e30f33d53fc8854f2fcc54193c84412149cfbc652bde2edd6e8a85
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv && {{ PYTHON }} -c 'import julia; julia.install()'"
+
+requirements:
+  build:
+    - python
+    - pip
+    - julia
+  host:
+    - python >=3.6,<4.0
+    - pip
+    - julia >=1.5.0
+  run:
+    - python >=3.6,<4.0
+    - {{ pin_compatible('julia') }}
```

Comment:
I think it's best to pin to >=1.7.1 here. What do you think @mkitti?

Reply:
Yes, probably. The 1.7 branch is the first minor version where we have implemented per-environment Julia depots.